### DISCUSSION TITLE
Can you define custom assets?

### DISCUSSION THREAD

**Discussion by dancb10** [fluttercandies/flutter_wechat_assets_picker#101] [Q&A] [answered]
Hello,
Is there a possibility of adding extra custom assets besides the video, audio and photos? I'm thinking about documents, archives and such
Thank you

**Comment by AlexV525** [comment, marked as answer]
Sure, delegates provide the ability that you can pick any types of assets you want. See https://github.com/fluttercandies/flutter_wechat_assets_picker/blob/master/example/lib/pages/custom_picker_page.dart#L297 .
